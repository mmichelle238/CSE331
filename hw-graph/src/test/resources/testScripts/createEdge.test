# Tests that a graph can be created, a pair of nodes and
# an edge between them can be created.

CreateGraph graph1
AddNode graph1 A
AddNode graph1 B
AddEdge graph1 A B 5