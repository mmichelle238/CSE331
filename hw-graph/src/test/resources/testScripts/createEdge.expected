# Tests that a graph can be created, a pair of nodes and
# an edge between them can be created.

created graph graph1
added node A to graph1
added node B to graph1
added edge 5 from A to B in graph1